wiki/PlayersManual.md

```diff
@@ -350,25 +350,35 @@ Trading panel: | &nbsp;
 :--- | :---
 Buy | +, Return, or Space
 Sell | -, Delete, or Backspace
-Buy / sell 5x | Shift
-Buy / sell 20x | Control / Command
-Buy / sell 500x | Option / Alt
 Buy / sell all | Shift + B / S
 View commodity price map | Map key
 
-The multiplier modifiers (Shift = 5x, Command / Control = 20x, and Option / Alt = 500x) also work in the hiring panel, the shipyard, and the outfitter. And, they can be combined together - for example, Shift + Control to buy or sell 100 items at a time.
+Multipliers: | &nbsp;
+:--- | :---
+5x | Shift
+20x | Control / Command
+500x | Option / Alt
+
+The multiplier modifiers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination - for example, Shift + Control to buy or sell 100 items at a time. All three will be a crazy whopping 50,000x.
 
 In addition to the multiplier keys, there are various other shortcuts in the outfitter:
 
-Outfitter: | &nbsp;
-:--- | :---
-Buy | B
-Sell | S
-Send to cargo | C
-Install | I
-Uninstall | U
+Outfitter: | &nbsp; | Priority 1 | Priority 2 | Priority 3
+:--- | :--- | :--- | :--- | :---
+Buy | B | Buy* from cargo to ship | Buy* from storage to ship | Buy from outfitter to ship
```

Comment:
Should this be `Buy*` Or should it just be `Install`?

Reply:
Yes, I think you're right, that makes it clearer.